.github/workflows/ci.yml

```diff
@@ -46,8 +52,6 @@ jobs:
       - name: Elixir dependencies
         if: steps.ex-deps-cache.outputs.cache-hit != 'true'
         run: |
-          mix local.rebar --force
-          mix local.hex --force
           mix deps.get
```

Comment:
Just to double check, in https://github.com/mbta/.github/pull/11 it adds `if: if: steps.deps-cache.outputs.cache-hit != 'true'` to `mix deps.get`—we're good with skipping that bit?

Reply:
We already have that here, see L53. (though thanks for discovering that `if: if:` typo in the other PR 😄)